scripts/benchmark_durations.py

```diff
@@ -67,6 +67,7 @@
     "test_runner/performance/test_copy.py::test_copy[neon]": 13.817,
     "test_runner/performance/test_copy.py::test_copy[vanilla]": 11.736,
     "test_runner/performance/test_gc_feedback.py::test_gc_feedback": 575.735,
+    "test_runner/performance/test_gc_feedback.py::test_gc_feedback_with_snapshots": 575.735,
```

Comment:
Is this from the CI run or just copy-pasted from the line above?

Reply:
@bayandin  would know where to get proper numbers

Reply:
yep that's copy-pasted

Reply:
Strictly speaking it's not necessary to update (because it's a fallback).

You can copy it from `get-benchmark-durations` job for the PR:
https://github.com/neondatabase/neon/actions/runs/10168523278/job/28123358074?pr=8474

Reply:
it seems that the newly-added test case is not in the list yet, and I'll update it later (probably it works after merging to main?)